lib/asciidoctor/pdf/formatted_text/parser.treetop

```diff
@@ -113,7 +113,7 @@ grammar Markup
   end
 
   rule character_hex
-    [0-9a-f] 2..5
+    [0-9A-Fa-f] 2..5
```

Comment:
Please put `a-f` before `A-F` (in all cases).

Reply:
ASCII ordering dies hard :)